invenio_administration/assets/semantic-ui/js/invenio_administration/src/actions/Actions.js

```diff
@@ -55,26 +62,54 @@ export class Actions extends Component {
             />
           )}
           {displayEdit && (
-            <Button
-              as="a"
-              disabled={disableEdit}
-              href={AdminUIRoutes.editView(listUIEndpoint, resource, idKeyPath)}
-              icon
-              labelPosition="left"
-            >
-              <Icon name="pencil" />
-              Edit
-            </Button>
+            <Popup
+              content={
+                disabledEditMessage
+                  ? disabledEditMessage
+                  : i18next.t("Resource is not editable.")
```

Comment:
Can you provide a better default explanation? As a user I already see that it is disabled :)
Maybe the text should be somehow injected?